app/controllers/todos_controller.rb

```diff
@@ -423,7 +423,13 @@ def update
     update_dependencies
     update_attributes_of_todo
 
-    @saved = @todo.save
+    begin
+      @saved = @todo.save!
+    rescue ActiveRecord::RecordInvalid => exception
+      @saved = false
+      exception.record.errors.each { |key| exception.record.errors[key].each { |value| @todo.errors[key] << value } }
```

Comment:
This line causes the value 'can't be blank' to be seen infinitely often when changing the description of an action to "". Could you explain how error messages are added to `@todo` and how this line should look like? I need to transport the error messages introduced in `dependency.rb:12` to `@todo` here, so that it is shown in the GUI.